src/hexen/p_enemy.c

```diff
@@ -4946,6 +4946,7 @@ void A_KoraxChase(mobj_t * actor)
             P_Teleport(actor, spot->x, spot->y, spot->angle, true);
         }
 
+        CheckACSPresent(249);
```

Comment:
Looking through the ACS files in Hexen and Deathkings, IDs 254 and 255 are used in almost every level to manage ambient sound effects and periodic enemy spawns, respectively. There's one other instance of 253 in the first level for ambient sound as well. The rest (249-252) are unused other than for Korax on its vanilla-exclusive map.

In general, I've noticed some usage patterns: the aforementioned 254/255, 100 sets constants (line specials, map variables, etc), and 200 manages deathmatch differences. The Korax map, however, throws out all of the above conventions. You could argue in favor of readable constants, but providing some commented context would do the same job seeing as how this usage is specific only to Korax itself. (Technically, a PWAD author could modify Korax's actions significantly using 249-254, so anything more descriptive than `KORAX_SCRIPT0_TID`, etc, may end up misleading.)